server/command_hooks.go

```diff
@@ -351,6 +351,21 @@ func getDialogWithSampleElements() model.Dialog {
 			Type:        "bool",
 			Placeholder: "Choice:",
 			HelpText:    "Pick a choice",
+		}, {
+			DisplayName: "Radio Option Selector",
+			Name:        "someradiooptionselector",
+			Type:        "radio",
```

Comment:
Would you please set a default element so we can use this for testing of this functionality works?
 ```suggestion
			Type:        "radio",
			Default:     "opt2",
```